src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceSpecifications.kt

```diff
@@ -22,6 +28,16 @@ data class LicenceQueryObject(
   val sortOrder: String? = null,
 )
 
+private val LICENCE_KINDS_WITH_RESPONSIBLE_COM: List<Class<out Licence>> = listOf(
```

Comment:
Keeping this list up to date is going to be difficult to remember and it is and always will be just all licence subtypes.. 

Think the easiest thing to do would be to add a new class field to `LicenceKind` and then build this list dynamically here: `LicenceKind.values().map{ it.clazz}`

Then we can't add a new kind without registering a class